src/ui/components/panels/loadout/active_loadout/dialogs/downtime/_BuySomeTime.vue

```diff
@@ -120,7 +120,7 @@ export default Vue.extend({
         nr.ResourceCost = 'Only a little time, and only if drastic measures are taken right now'
       else if (this.skillRoll < 20)
         nr.ResourceCost = 'The situation becomes precarious or desperate'
-      this.pilot.Reserves.push(nr)
+      this.pilot.AddReserve(nr)
```

Comment:
Noticed that after adding a reserve from Downtime, if I immediately refreshed, the reserve would disappear in many cases.  Replaced all instances of "pushing" a reserve or organization with the appropriate function from Pilot.ts.